R/ncdf.R

```diff
@@ -368,7 +396,7 @@ read_ncdf = function(.x, ..., var = NULL, ncsub = NULL, curvilinear = character(
 
     if(length(auxiliary_x) > 0 && length(auxiliary_y) > 0 &&
        auxiliary_x %in% c_v$X && auxiliary_y %in% c_v$Y &
-       !is.na(prj)) {
+       has_prj) {
```

Comment:
I don't know what the prj is yet -- but I can at least check if there is one at all. This might be kinda flakey but it works for the purpose.